lib/jsdom/living/hr-time/Performance-impl.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const EventTargetImpl = require("../events/EventTarget-impl").implementation;
+
+class PerformanceImpl extends EventTargetImpl {
+  constructor(args, privateData) {
+    super(args, privateData);
+
+    this._performance = privateData.performanceData;
+  }
+
+  now() {
+    return this._performance.now();
+  }
+
+  get timeOrigin() {
+    return this._performance.timeOrigin;
+  }
+
+  toJSON() {
+    return this._performance.toJSON();
```

Comment:
Shouldn't this return the timeOrigin? That's what [Default] would imply. (One day we can implement [Default] in webidl2js.)

Reply:
Seems like Timothy looked into this: https://github.com/jsdom/w3c-hr-time/blob/master/lib/performance.js#L46

Reply:
Hmm seems like either a spec bug or a Firefox bug, interesting.